docker/Ubuntu.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:22.04
+FROM --platform=linux/amd64 ubuntu:22.04
```

Comment:
The `--platform` flag must be place before the the image argument, or else it will fail with:

```
Ubuntu.Dockerfile:1
--------------------
   1 | >>> FROM ubuntu:22.04 --platform=linux/amd64
   2 |
   3 |     # Install dependecies
--------------------
ERROR: failed to solve: dockerfile parse error on line 1: FROM requires either one or three arguments
```